common/log.c

```diff
@@ -592,7 +592,7 @@ log_message(const enum logLevels lvl, const char *msg, ...)
         /* log to syslog*/
         /* %s fix compiler warning 'not a string literal' */
         syslog(internal_log_xrdp2syslog(lvl), "(%d)(%ld)%s", g_getpid(),
-               tc_get_threadid(), buff + 20);
+               (long) tc_get_threadid(), buff + 20);
```

Comment:
Alright, this is OK for now. Anybody changing to the thread ID type would change tc_get_threadid() and would need to recheck the callers, including this one.